grails-app/i18n/messages.properties

```diff
@@ -3191,6 +3191,7 @@ stockMovement.dateReceived.label=Date Received
 stockMovement.dateIssued.label=Date Issued
 stockMovement.dateApproved.label=Date Approved
 stockMovement.dateRejected.label=Date Rejected
+stockMovement.emptyTable.label=No items to display
```

Comment:
It won't work. For the react side we are using `react.stockMovement...`. On the react side we are fetching all of those translations that start with `react.stockMovement` when we are accessing this workflow.
\+ We have to call the hook for fetching translations, did you use it?